instrumentation/github.com/aws/aws-sdk-go-v2/otelaws/s3attributes.go

```diff
@@ -0,0 +1,411 @@
+
+		s3Attributes = append(s3Attributes, semconv.AWSS3Key(aws.ToString(v.Key)))
+
+	case *s3.PutObjectLegalHoldInput:
+		s3Attributes = append(s3Attributes, semconv.AWSS3Bucket(aws.ToString(v.Bucket)))
+
+		s3Attributes = append(s3Attributes, semconv.AWSS3Key(aws.ToString(v.Key)))
+
+	case *s3.PutObjectLockConfigurationInput:
+		s3Attributes = append(s3Attributes, semconv.AWSS3Bucket(aws.ToString(v.Bucket)))
+
+	case *s3.PutObjectRetentionInput:
+		s3Attributes = append(s3Attributes, semconv.AWSS3Bucket(aws.ToString(v.Bucket)))
+
+		s3Attributes = append(s3Attributes, semconv.AWSS3Key(aws.ToString(v.Key)))
+
+	case *s3.PutObjectTaggingInput:
+		s3Attributes = append(s3Attributes, semconv.AWSS3Bucket(aws.ToString(v.Bucket)))
+
+		s3Attributes = append(s3Attributes, semconv.AWSS3Key(aws.ToString(v.Key)))
+
+	case *s3.PutPublicAccessBlockInput:
+		s3Attributes = append(s3Attributes, semconv.AWSS3Bucket(aws.ToString(v.Bucket)))
+
+	case *s3.RestoreObjectInput:
+		s3Attributes = append(s3Attributes, semconv.AWSS3Bucket(aws.ToString(v.Bucket)))
+
+		s3Attributes = append(s3Attributes, semconv.AWSS3Key(aws.ToString(v.Key)))
+
+	case *s3.SelectObjectContentInput:
+		s3Attributes = append(s3Attributes, semconv.AWSS3Bucket(aws.ToString(v.Bucket)))
+
+		s3Attributes = append(s3Attributes, semconv.AWSS3Key(aws.ToString(v.Key)))
+
+	case *s3.UploadPartInput:
+		s3Attributes = append(s3Attributes, semconv.AWSS3Bucket(aws.ToString(v.Bucket)))
+
+		s3Attributes = append(s3Attributes, semconv.AWSS3Key(aws.ToString(v.Key)))
+
+		s3Attributes = append(s3Attributes, semconv.AWSS3PartNumber(int(v.PartNumber)))
+
+		s3Attributes = append(s3Attributes, semconv.AWSS3UploadID(aws.ToString(v.UploadId)))
+
+	case *s3.UploadPartCopyInput:
+		s3Attributes = append(s3Attributes, semconv.AWSS3Bucket(aws.ToString(v.Bucket)))
+
+		s3Attributes = append(s3Attributes, semconv.AWSS3Key(aws.ToString(v.Key)))
+
+		s3Attributes = append(s3Attributes, semconv.AWSS3PartNumber(int(v.PartNumber)))
+
+		s3Attributes = append(s3Attributes, semconv.AWSS3UploadID(aws.ToString(v.UploadId)))
+
+	case *s3.WriteGetObjectResponseInput:
+		// WriteGetObjectResponseInput defines no attributes
+	}
+
+	return s3Attributes
+}
+
+// s3ShorthandSerializeDelete serializes the Delete struct into shorthand syntax
+// https://docs.aws.amazon.com/cli/latest/userguide/cli-usage-shorthand.html
+func s3ShorthandSerializeDelete(d *s3types.Delete) string {
+	var builder strings.Builder
+
+	fmt.Fprint(&builder, "Objects=[")
+	count := len(d.Objects)
+	for i, object := range d.Objects {
+		fmt.Fprint(&builder, "{")
+
+		fmt.Fprintf(&builder, "Key=%s", aws.ToString(object.Key))
+
+		if object.VersionId != nil {
+			fmt.Fprintf(&builder, ",VersionId=%s", aws.ToString(object.VersionId))
+		}
+
+		fmt.Fprint(&builder, "}")
+		if i+1 != count {
+			fmt.Fprint(&builder, ",")
+		}
+	}
+	fmt.Fprint(&builder, "],")
+
+	fmt.Fprintf(&builder, "Quiet=%t", d.Quiet)
+
```

Comment:
```suggestion
	fmt.Fprint(&builder, "Objects=[")
	count := len(d.Objects)
	for i, object := range d.Objects {
		fmt.Fprint(&builder, "{")
		fmt.Fprintf(&builder, "Key=%s", aws.ToString(object.Key))
		if object.VersionId != nil {
			fmt.Fprintf(&builder, ",VersionId=%s", aws.ToString(object.VersionId))
		}
		fmt.Fprint(&builder, "}")
		if i+1 != count {
			fmt.Fprint(&builder, ",")
		}
	}
	fmt.Fprint(&builder, "],")
	fmt.Fprintf(&builder, "Quiet=%t", d.Quiet)
```
invalid blank lines should be removed.